src/Components/DataEntry/TextField.vue

```diff
@@ -0,0 +1,250 @@
+<template>
+	<div class="c-application c-textfield" :class="[computedLabelStyle]">
+		<input
+			:id="computedId"
+			class="c-textfield--input"
+			:value="value"
+			:type="type"
+			:placeholder="placeholder"
+			:name="computedId"
+			:label="label"
+			:align="align"
+			:focus="focus"
+			:readonly="readOnly"
+			:disabled="disabled"
+			:error="error"
+			:style="[computedAlign]"
+			:class="[computedLined, computedError]"
+			v-bind="$attrs"
+			@input="$emit('input', $event.target.value)"
+			v-on="$listeners"
+		/>
+		<label v-if="label !== ''" :for="computedId" class="c-textfield--label">{{ label }}</label>
+		<Typography v-if="error" type="caption2" color="red400" element="p" class="c-textfield--message">
+			<slot name="errorMessage" />
+		</Typography>
+	</div>
+</template>
+
+<script>
+import Typography from '@/src/Elements/Core/Typography/Typography';
+export const textfieldTypes = ['text', 'number', 'password', 'email', 'tel', 'url'];
+export const textfieldAligns = ['left', 'center', 'right'];
+
+export default {
+	name: 'TextField',
+	inheritAttrs: false,
+	props: {
+		type: {
+			type: String,
+			default: 'text',
+			validator(value) {
+				const isValid = textfieldTypes.indexOf(value) !== -1;
+				if (!isValid) {
+					console.error(`${value} is not a valid name of the textfield type`);
+				}
+				return isValid;
+			},
+		},
+		value: {
+			type: String,
+			default: '',
+			validator(value) {
+				return typeof value === 'string';
+			},
+		},
+		placeholder: {
+			type: String,
+			default: '내용을 입력하세요.',
+			validator(value) {
+				return typeof value === 'string';
+			},
+		},
+		label: {
+			type: String,
+			default: '',
+			validator(value) {
+				return typeof value === 'string';
+			},
+		},
+		filled: {
+			type: Boolean,
+			default: false,
+			validator(value) {
+				return typeof value === 'boolean';
+			},
+		},
+		align: {
+			type: String,
+			default: 'left',
+			validator(value) {
+				const isValid = textfieldAligns.indexOf(value) !== -1;
+				if (!isValid) {
+					console.error(`${value} is not a valid name of the textfield align`);
+				}
+				return isValid;
+			},
+		},
+		focus: {
+			type: Boolean,
+			default: false,
+			validator(value) {
+				return typeof value === 'boolean';
+			},
+		},
+		readOnly: {
```

Comment:
```suggestion
		readonly: {
```
html input 태그 속성이 readonly니까, 기존 속성명을 따르면 더 직관적일 것 같은데 어떤가요?

Reply:
readonly로 하면 속성이 안먹는거 같아요.

Reply:
오잉? 저는 해보니까 잘 돼요. mdx에서 :read-only="readonly"로 되어있다거나하는 오타를 예상해봅니다

Reply:
!!! 수정 완료 :)